### DISCUSSION TITLE
Export from Cvxpy to Sympy

### DISCUSSION THREAD

**Discussion by iamharshit** [cvxpy/cvxpy#1726] [Q&A]
Is it possible to export Cvxpy's optimization problem (i.e. Objective equation and Constraint expressions) to Sympy?

**Comment by phschiele** [comment]
This question also came up on Discord recently. 
Short answer: CVXPY is not usage-compatible with SymPy.

**Reply by iamharshit** [reply]
Thanks for the info.
I should be joining the discord then, could you share the link of that conversation.
A follow up - If Cvxpy doesn't support it out of the box, is it possible to manually write codes to achieve that (a template would be very helpful)?

**Reply by phschiele** [reply]
Here is the link to the discord question: https://discord.com/channels/845323972962549790/845323972962549794/903819044652744744
@rileyjmurray has answered the question there, but it was more about the differences between SymPy and SciPy.

I'm not aware of any previous work to export to SymPy (also looked through the issues).
